google-cloud-platform/index.md

```diff
@@ -9,7 +9,7 @@ has-permalinks: false
 # {{ page.title }} <img src="images/GoogleCloudPlatform-logo.png" alt="logo for Google Cloud Platform">
 
 **Note**: There are many ways to host a Dart server. You might also be
-interested in [Heroku][] and [Dartvoid][].
+interested in [Heroku][] and [Sourcevoid](https://www.sourcevoid.com/).
```

Comment:
I'm sorry for the confusion on my part, I'm not used to the "[name][]" style links, I didn't know about that syntax variation. Now the links use the same style as before but with the updated name and url.